neuraxle/metaopt/auto_ml.py

```diff
@@ -479,14 +429,11 @@ def __len__(self):
         return len(self.trials)
 
 
-class AutoMLSequentialWrapper(NonTransformableMixin, MetaStepMixin, BaseStep):
+class AutoMLSequentialWrapper(MetaStepMixin, BaseStep):
```

Comment:
I think you forgot to add `HandlerMixin` here because in init you use that handlermixin?